plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/server/OpenShiftServerEditorSection.java

```diff
@@ -762,13 +774,31 @@ public boolean isCanceled() {
 
 	@Override
 	public IStatus[] getSaveStatus() {
-		return new IStatus[] { formPresenterSupport.getCurrentStatus() };
+		return new IStatus[] { getFormPresenterStatus() };
+	}
+
+	private IStatus getFormPresenterStatus() {
+		IStatus saveStatus = Status.OK_STATUS;
+		if (formPresenterSupport != null
```

Comment:
i don't think that `formPresenterSupport` can be `null`.
If i miss something, then these null checks must be also added to `dispose` method